test/test-network/conf/25-qdisc-ingress-netem-compat.network

```diff
@@ -0,0 +1,16 @@
+[Match]
+Name=test1
+
+[Network]
+IPv6AcceptRA=no
+Address=10.1.2.4/16
+
+[TrafficControlQueueingDiscipline]
+Parent=root
+NetworkEmulatorDelaySec=50ms
+NetworkEmulatorDelayJitterSec=10ms
+NetworkEmulatorLossRate=20%
+NetworkEmulatorPacketLimit=100
+
+[QDisc]
```

Comment:
Can we split the test case . I think this should be different we should not club it together.

Reply:
I do not mind that, but why? Two sections should not interfere each other.

Reply:
This works because `root=` is different. 
```
[TrafficControlQueueingDiscipline]
Parent=root

[QDisc]
Parent=ingress
```

Will not work and not backward compatible.  It should replicate the same behaviour. We just added a new feature which is just fine .

Reply:
Still I do not catch the meaning, sorry. But adding more tests are generally ok. I will add more tests.

Reply:
```
[TrafficControlQueueingDiscipline]
Parent=root
```
Does this work before? What is the expected result?

Reply:
The same happens for `tc` command:
```
$ sudo tc qdisc add dev hoge root
Error: Specified qdisc not found.
```

Reply:
yes it does. If you go back to the commit .  It's not backward compatibility rather a feature.  
Please see here https://github.com/systemd/systemd/pull/14584.

Reply:
Please provide the relevant `tc` command then. If I understand correctly, `root` must be combined  with other kind of qdisc.

Reply:
did you tested and verified what I  asked?

Reply:
Yes. Before #14767, 
```
[TrafficControlQueueingDiscipline]
Parent=root
```
does not work.